requirements.txt

```diff
@@ -2,3 +2,9 @@ zipp>=0.5; python_version < "3.8"
 typing-extensions>=3.6.4; python_version < "3.8"
 pytest>=4.6.5
 mypy
+numpy
+pydantic
+types-dataclasses
+typing_extensions
+wasabi
```

Comment:
Likewise, as far as I can see `wasabi` is only required to format an error message in a table once...

Reply:
Yeah, that's true. Is vendoring the `table` function an option for us?

Reply:
If it's easy to just rewrite that error message without `table` I'd vote for that. Otherwise I guess `wasabi` is pretty lightweight itself so perhaps it's no big deal to keep it. I'm not sure what you mean by vendoring - `wasabi` is already maintained by us?

Reply:
I meant to copy-paste the `table` function and functions called from there directly into `catalogue`, so we wouldn't have to include `wasabi` as dependency. I'll have a look a rewriting the error message.

Reply:
All the error messages formatted by `table()` in the tests were of the pattern `[faulty part of the config, error description]`, for example `[('a.b -> foo', 'uses section variable in string or list')]`, to `"a.b -> foo      uses section variable in string or list"`. If we only want to do this, then dropping `wasabi` is trivial. Do we expect any more complex error messages? 

Otherwise I'd just print one tab-separated error per line and remove `wasabi` from the dependencies.